producer/murmur2.go

```diff
@@ -0,0 +1,124 @@
+// This file was copied from https://github.com/burdiyan/kafkautil/blob/3e3bfeae0ffaf3b3d9b431463d9e58f840173188/partitioner.go
```

Comment:
🌷 No tests?

Reply:
Oh good point, I forgot to copy them. Thanks for noticing! Added in 9c674a8bbae1a30c549192fdb502f401bef5c675